model/map/src/main/java/org/keycloak/models/map/realm/MapRealmEntity.java

```diff
@@ -1024,7 +1025,7 @@ public boolean removeClientInitialAccess(String id) {
         return removed;
     }
 
-    public Stream<MapClientInitialAccessEntity> getClientInitialAccesses() {
-        return clientInitialAccesses.values().stream();
+    public Collection<MapClientInitialAccessEntity> getClientInitialAccesses() {
```

Comment:
Is this change necessary? The `Collection` return type is the only one in the entities so it seem inconsistent with the rest of the cide.

Reply:
This is because we don't have [exists](https://github.com/keycloak/keycloak/pull/7931/files#diff-f785fd9ed631f0d3e1e9917ed2a9fdd4d900baa992038ce9afa7fdf2c7f503a4R113) and [not-exists](https://github.com/keycloak/keycloak/pull/7931/files#diff-f785fd9ed631f0d3e1e9917ed2a9fdd4d900baa992038ce9afa7fdf2c7f503a4R129) methods working with Stream, only with collections.

We have a `Collection` returned in many places across all entities, but we usually use `Set` or `List`. Do we want to refactor it and use `Stream`s everywhere?

Reply:
You're right, we can keep it changed as is here for now